components/badge/badge.spec.ts

```diff
@@ -237,3 +239,117 @@ export class NzTestBadgeRtlComponent {
   direction: Direction = 'rtl';
   count = 5;
 }
+
+describe('badge', () => {
+  let fixture: ComponentFixture<NzBadgeComponent>;
+  let component: NzBadgeComponent;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
```

Comment:
`NzBadgeComponent` is a standalone component, no need to import here, use `TestBed.createComponent()` directly